### DISCUSSION TITLE
MicroWin Script Update

### DISCUSSION THREAD

**Discussion by oiproc5** [ChrisTitusTech/winutil#2831] [Ideas]
would it be possible to include scripts into MicroWin that allow you to include certain files or directories into the iso so that they will be for instance on your desktop or in your documents folder?

also the option to install the applications that winutil offers during windows setup.. its possible with NTLite, i hope someone is capable of implementing this into winutil, this project is a lifesaver

**Comment by FatBastard0** [comment]
Yeah I was thinking that maybe an toggle our button for an expert mode
I do agree with Mr. Titus, with the concept that MicrWin, should be very easy to you. 
Me being new at game, I am that guy that would get lost on how to do something.
